src/firejail/main.c

```diff
@@ -1490,7 +1490,7 @@ int main(int argc, char **argv, char **envp) {
 		else if (strncmp(argv[i], "--rlimit-fsize=", 15) == 0) {
 			cfg.rlimit_fsize = parse_arg_size(argv[i] + 15);
 			if ( cfg.rlimit_fsize == 0 ) {
-				perror("Error: given rlimit-size is invalid. use only non-negative numbers and k,m,g suffix for size");
+				perror("Error: invalid rlimit-fsize. only use positive numbers and k, m or g suffix.");
```

Comment:
It's a capital O (not o) after completing a sentence:
perror("Error: invalid rlimit-fsize. Only use positive numbers and k, m or g suffix.");
Please change this and we're set for merge.